### PR TITLE
Fix bad fields in 7.x dashboards

### LINKED ISSUE #11: [Dashboards 7.x] Problems loading visualizations for Services, Errors and Transactions dashboards
I haven't problems on some of the 7.x dashboards, where certain visualizations don't load.

#### Services

screencapture-localhost-5601-doi-app-kibana-2019-02-28-13_00_19

#### Errors

screencapture-localhost-5601-doi-app-kibana-2019-02-28-13_00_27

#### Transactions

screencapture-localhost-5601-doi-app-kibana-2019-02-28-13_00_09

cc @bmorelli25 @graphaelli

### PR DESCRIPTION
**Pull request by bmorelli25**
Fixes #11 

**transactions.json**
`transaction.name.keyword` -> `transaction.name`
screen shot 2019-02-28 at 12 04 02 pm

---

**errors.json**
`view error` -> `error.culprit` & removed other `error.culprit` (had to export this one)
screen shot 2019-02-28 at 11 58 53 am

---

**services.json**
removed `view errors`. I didn't end up replacing it with anything because nothing looked quite right. Open to recommendations.
screen shot 2019-02-28 at 11 45 17 am

### CODE REVIEW

File: apm-ui/7.x/errors.json [hunk 1]
```diff
@@ -33,27 +33,27 @@
...
+    },
     "_migrationVersion": {
       "visualization": "7.0.0"
     },
     "_references": [
       {
-        "id": "apm-*",
         "name": "kibanaSavedObjectMeta.searchSourceJSON.index",
-        "type": "index-pattern"
+        "type": "index-pattern",
```
Comment (graphaelli) [inline apm-ui/7.x/errors.json]: let's keep these alphabetized to minimize the diffs

File: apm-ui/7.x/transactions.json [hunk 2]
```diff
@@ -56,7 +56,7 @@
...
...ll},\"perPage\":25,\"showPartialRows\":false,\"totalFunc\":\"sum\",\"showTotal\":false,\"showMeticsAtAllLevels\":false},\"aggs\":[{\"params\":{\"orderBy\":\"1\",\"field\":\"transaction.name\",\"customLabel\":\"Transaction\",\"order\":\"desc\",\"size\":1000},\"type\":\"terms\",\"enabled\":true,\"id\":\"2\",\"schema\":\"bucket\"},{\"params\":{\"sortField\":\"@timestamp\",\"customLabel\":\"Type\",\"field\":\"transaction.type\",\"sortOrder\":\"desc\",\"aggregate\":\"concat\",\"size\":1},\"type\":\"top_hits\",\"enabled\":true,\"id\":\"5\",\"schema\":\"metric\"},{\"params\":{\"field\":\"transaction.duration.us\",\"customLabel\":\"Avg. Resp Time (ms)\"},\"type\":\"avg\",\"enabled\":true,\"id\":\"1\",\"schema\":\"metric\"},{\"params\":{\"field\":\"transaction.duration.us\",\"customLabel\":\"Resp Time (ms)\",\"percents\":[95]},\"type\":\"percentiles\",\"enabled\":true,\"id\":\"3\",\"schema\":\"metric\"},{\"params\":{\"sortField\":\"@timestamp\",\"customLabel\":\"View Spans\",\"field\":\"transaction.id\",\"sortOrder\":\"desc\",\"aggregate\":\"concat\",\"size\":1},\"type\":\"top_hits\",\"enabled\":true,\"id\":\"4\",\"schema\":\"metric\"}],\"title\":\"Top Transactions for Time Period [APM]\"}"
```
Comment (graphaelli) [inline apm-ui/7.x/transactions.json]: 👍

File: apm-ui/7.x/services.json [hunk 3]
```diff
@@ -56,7 +56,7 @@
...
+      "visState": "{\"params\":{\"sort\":{\"columnIndex\":null,\"direction\":null},\"perPage\":10,\"showPartialRows\":false,\"totalFunc\":\"sum\",\"showTotal\":false,\"showMeticsAtAllLevels\":false},\"type\":\"table\",\"aggs\":[{\"type\":\"avg\",\"enabled\":true,\"id\":\"1\",\"params\":{\"field\":\"transaction.duration.us\",\"customLabel\":\"Avg. Trans. Time\"},\"schema\":\"metric\"},{\"type\":\"percentiles\",\"enabled\":true,\"id\":\"3\",\"params\":{\"field\":\"transaction.duration.us\",\"percents\":[95],\"customLabel\":\"Trans. Time\"},\"schema\":\"metric\"},{\"type\":\"cardinality\",\"enabled\":true,\"id\":\"4\",\"params\":{\"field\":\"transaction.id\",\"customLabel\":\"Total Transactions\"},\"schema\":\"metric\"},{\"type\":\"cardinality\",\"enabled\":true,\"id\":\"6\",\"params\":{\"field\":\"error.id\",\"customLabel\":\"Errors\"},\"schema\":\"metric\"},{\"type\":\"terms\",\"enabled\":true,\"id\":\"2\",\"params\":{\"orderBy\":\"1\",\"field\":\"service.name\",\"order\":\"desc\",\"size\":1000},\"schema\":\"bucket\"}],\"title\":\"Services [APM]\"}"
```
Comment (graphaelli) [inline apm-ui/7.x/services.json]: 👍

File: apm-ui/7.x/errors.json (hunk 1 above)
Comment (bmorelli25) [reply apm-ui/7.x/errors.json]: Good point

**Pull request merged**